ctfdpy/ctfdpy/http.py

```diff
@@ -39,22 +39,85 @@ def _request(
         params: dict[str, Any] | None = None,
         json: dict[str, Any] | None = None,
     ) -> dict[str, Any] | None:
-        try:
-            response = self.client.request(
-                method=method.value, url=endpoint, params=params, json=json
-            )
+        """
```

Comment:
Theses docstrings seems to follow a custom version of numpy flavor docstring. It would be preferable to conform to a common convention. Google docstrings  looks good to me and is the most widely used. :
- https://google.github.io/styleguide/pyguide.html#383-functions-and-methods
- https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
The use of autoDocstring extension setted to create google doctring could be a good helper for the future :
- https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring